src/Umbraco.Web.UI.Client/src/packages/media/media/components/input-media/input-media.context.ts

```diff
@@ -19,7 +18,7 @@ export class UmbMediaPickerInputContext extends UmbPickerInputContext<
 	UmbMediaPickerModalValue
 > {
 	constructor(host: UmbControllerHost) {
-		super(host, UMB_MEDIA_ITEM_REPOSITORY_ALIAS, UMB_MEDIA_PICKER_MODAL);
+		super(host, UMB_MEDIA_ITEM_REPOSITORY_ALIAS, UMB_MEDIA_TREE_PICKER_MODAL);
```

Comment:
Hi @NguyenThuyLan, we should not switch all media inputs to tree pickers. Instead we need to look at the specific implementation in the user group workspace, only switch out that implementation to use a tree picker ux.